.gitignore

```diff
@@ -35,6 +35,7 @@ src/app/static/public/css
 src/app/static/public/js
 demo
 git
+!src/app/static/src/**/*
```

Comment:
to make sure that directories named `git` in the static app src are not excluded by the previous line!